src/reporting-config/__tests__/reporting-config-slice.test.ts

```diff
@@ -1,14 +1,369 @@
-import { normalizeReportingConfig } from '../reporting-config-parsers';
```

Comment:
This is effectively a new test file with a correct name!

This test file is to help cover some of the gritty nuance when we are collapsing tasks across several levels of the JSON.